src/core/ConfigManager.cpp

```diff
@@ -131,17 +140,17 @@ void ConfigManager::upgrade()
 		return;
 	}
 
-	ProjectVersion createdWith = m_version;
-	
-	if (createdWith.setCompareType(ProjectVersion::Build) < "1.1.90")
-	{
-		upgrade_1_1_90();
-	}
+	using upgradeMethod = void(ConfigManager::*)();
```

Comment:
According to our coding conventions, types must start with an uppercase letter. Also, this line can be moved into the header, so it can be used in the declaration of `m_upgradeMethods` too.